src/SFA.DAS.Apprenticeships.Web/Extensions/UiExtensions.cs

```diff
@@ -5,6 +5,11 @@ namespace SFA.DAS.Apprenticeships.Web.Extensions
 {
     public static class UiExtensions
     {
+        public static string FormatDate(this DateTime date)
```

Comment:
I think there should already be GDS standard extensions for this (e.g. .ToGdsDate, .ToGdsFormatDate etc.)

Reply:
nvm this is actually in commitments only